README.md

```diff
@@ -1,2 +1,39 @@
 # kotlinx-benchmark-table-action
 An Action to take info from kotlinx-benchmark outputs and display them in a table as a pull request comment
+
+Check [action.yml](action.yml) for a description of inputs and outputs.
+
+### Configuration sample
+
+1. As a GitHub Summary check:
+
+```yml
+- name: Create Benchmark Table
+  id: benchmark-table
+  uses: boswelja/kotlinx-benchmark-table-action@0.0.2
+  with:
+    benchmark-results: ./benchmarks/build/reports/benchmarks/main/**/jvm.json
+- name: Post Results
+  run: |
+    echo '## Benchmarks results ⏱️' >> $GITHUB_STEP_SUMMARY
+    echo '${{steps.benchmark-table.outputs.benchmark-table}}️' >> $GITHUB_STEP_SUMMARY
+```
+
+2. As a GitHub PR comment (Assuming the action is running on a GitHub runners):
```

Comment:
> (Assuming the action is running on a GitHub runners)

Is there ever a case where it's not?

Reply:
Yes, there is one case that I'm actually facing, i.e., when you declare in the action to use a self-hosted container, in that case the action does not run in the GH host machine, so won't have the GH CLI tool.

https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container

Reply:
Maybe we can rephrase it better to make it more clear 🤔

Reply:
That's interesting! I had no idea self-hosted runners didn't have the necessary tools built-in